src/CoreShop2VueStorefrontBundle/Bridge/DocumentMapper/AbstractMapper.php

```diff
@@ -34,6 +32,6 @@ abstract class AbstractMapper
 
     protected function getDateTime(int $date): string
     {
-        return Carbon::createFromTimestamp($date)->format('Y-m-d H:i:s');
+        return Carbon::createFromTimestamp($date)->format(\DateTimeInterface::RFC3339);
```

Comment:
Elasticsearch will reject dates in `Y-m-d H:i:s` format, space is not an allowed separator.

Reply:
👍